samples/MicroServices/NBB.Contracts/NBB.Contracts.Application/CommandHandlers/ContractCommandHandlers.cs

```diff
@@ -16,25 +16,25 @@ public class ContractCommandHandlers :
 
         public ContractCommandHandlers(IEventSourcedRepository<Contract> repository)
         {
-            this._repository = repository;
+            _repository = repository;
         }
 
-        public async Task Handle(CreateContract command, CancellationToken cancellationToken)
+        public async Task Handle(CreateContract request, CancellationToken cancellationToken)
```

Comment:
It seams a little odd to name commands as "request" even if they are in the implementation of IRequestHandler